docs/documentation/head/prepare.md

```diff
@@ -9,17 +9,14 @@ nexttitle: Creating a Database
 next: your-database.html
 ---
 
-Because Java does not support using unix sockets the PostgreSQL™ server must be
+Generally speaking Java does not support using unix sockets the PostgreSQL™ server must be
```

Comment:
That doesn't read quite right with "*Generally speaking*" unless you add a "*so* the PostgreSQL...".  I'd stick with "*Because*" or maybe "*As*"

Reply:
Well the driver does support local sockets now though

Reply:
I meant that the revised sentence is not grammatically correct without adding a "*so*".

Is the intention to say that it's not supported out of the box, but you can make it work with third party code / custom socket factories? If so how about:

```
Out of the box, Java does not support unix sockets so the PostgreSQL server must be configured to allow TCP/IP connections.
```

Could also add a follow up note linking to the recommended unix sockets SocketFactory in the connect docs. Something like:

```
Unix sockets can be used with the driver in conjunction with third-party `javax.net.SocketFactory` implementations. See ...link-to-connect-docs-goes-here... for an example configuration.
```